docs/AboutUs.md

```diff
@@ -46,7 +46,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](https://www.linkedin.com/in/tang-jun-kiat-0540582b6/)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Deliverables and deadlines
 
 ### Prakamya Singh
```

Comment:
Make my responsibilities as `Commands, Code Quality` and role as `QA (Quality Assurance)`